### PR TITLE
Add and fix ALB related functions

### PR DESCRIPTION
**Pull request by szombi**
- add `Deprecated` comment for the unsupported functions
- add missing functions and unit tests
- add missing alb examples

### CODE REVIEW

File: api/container/containerv2/alb.go [hunk 1]
```diff
@@ -88,10 +193,108 @@ func (r *alb) GetAlb(albID string, target ClusterTargetHeader) (AlbConfig, error
 	return successV, err
 }
 
+func (r *alb) UpdateAlb(updateAlbReq UpdateALBReq, target ClusterTargetHeader) error {
```
Comment (attilatabori) [inline api/container/containerv2/alb.go:197]: This func may miss a comment/doc. Also in the example there is a call for ChangeALBUpdatePolicy to disable auto update. Is it mandatory? should this be done here as well?

File: examples/container/V2containers/AddIgnoredIngressStatusErrorsV2/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/AddIgnoredIngressStatusErrorsV2/README.md:1]: AddIgnoredIngressStatusErrors example
cpy paste error ?

File: examples/container/V2containers/GetALBAutoscaleConfigurationV2/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/GetALBAutoscaleConfigurationV2/README.md:1]: cpy paste error ?

File: examples/container/V2containers/GetAlbClusterHealthCheckConfigV2/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/GetAlbClusterHealthCheckConfigV2/README.md:1]: cpy paste error ?

File: examples/container/V2containers/GetIgnoredIngressStatusErrorsV2/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/GetIgnoredIngressStatusErrorsV2/README.md:1]: cpy paste error?

File: examples/container/V2containers/GetIngressLoadBalancerConfigV2/README.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/GetIngressLoadBalancerConfigV2/README.md:1]: cpy paste

File: examples/container/V2containers/GetIngressStatusV2/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/GetIngressStatusV2/README.md:1]: cpy paste

File: examples/container/V2containers/ListAlbImagesV2/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/ListAlbImagesV2/README.md:1]: cpy paste

File: examples/container/V2containers/RemoveALBAutoscaleConfigurationV2/README.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/RemoveALBAutoscaleConfigurationV2/README.md:1]: cpy paste

File: examples/container/V2containers/RemoveIgnoredIngressStatusErrorsV2/README.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/RemoveIgnoredIngressStatusErrorsV2/README.md:1]: cpy paste error

File: examples/container/V2containers/SetALBAutoscaleConfigurationV2/README.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/SetALBAutoscaleConfigurationV2/README.md:1]: cpy paste error

File: examples/container/V2containers/SetAlbClusterHealthCheckConfigV2/README.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/SetAlbClusterHealthCheckConfigV2/README.md:1]: cpy paste error

File: examples/container/V2containers/SetIngressStatusStateV2/README.md [hunk 13]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/SetIngressStatusStateV2/README.md:1]: cpy paste error

File: examples/container/V2containers/UpdateAlbV2/README.md [hunk 14]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/UpdateAlbV2/README.md:1]: cpy paste

File: examples/container/V2containers/UpdateIngressLoadBalancerConfigV2/README.md [hunk 15]
```diff
@@ -0,0 +1,9 @@
+# ChangeALBUpdatePolicy example
```
Comment (attilatabori) [inline examples/container/V2containers/UpdateIngressLoadBalancerConfigV2/README.md:1]: cpy paste

**Review by attilatabori — COMMENTED** [review]
In general, structs and methods could be ordered alphabetically

File: api/container/containerv2/alb.go (hunk 1 above)
Comment (szombi) [reply api/container/containerv2/alb.go:202]: Fixed

File: examples/container/V2containers/UpdateIngressLoadBalancerConfigV2/README.md (hunk 15 above)
Comment (szombi) [reply examples/container/V2containers/UpdateIngressLoadBalancerConfigV2/README.md]: Fixed

File: examples/container/V2containers/SetIngressStatusStateV2/README.md (hunk 13 above)
Comment (szombi) [reply examples/container/V2containers/SetIngressStatusStateV2/README.md]: Fixed

File: examples/container/V2containers/SetAlbClusterHealthCheckConfigV2/README.md (hunk 12 above)
Comment (szombi) [reply examples/container/V2containers/SetAlbClusterHealthCheckConfigV2/README.md]: Fixed

File: examples/container/V2containers/SetALBAutoscaleConfigurationV2/README.md (hunk 11 above)
Comment (szombi) [reply examples/container/V2containers/SetALBAutoscaleConfigurationV2/README.md]: Fixed

File: examples/container/V2containers/RemoveIgnoredIngressStatusErrorsV2/README.md (hunk 10 above)
Comment (szombi) [reply examples/container/V2containers/RemoveIgnoredIngressStatusErrorsV2/README.md]: Fixed

File: examples/container/V2containers/RemoveALBAutoscaleConfigurationV2/README.md (hunk 9 above)
Comment (szombi) [reply examples/container/V2containers/RemoveALBAutoscaleConfigurationV2/README.md]: Fixed

File: examples/container/V2containers/ListAlbImagesV2/README.md (hunk 8 above)
Comment (szombi) [reply examples/container/V2containers/ListAlbImagesV2/README.md]: Fixed

File: examples/container/V2containers/GetIngressStatusV2/README.md (hunk 7 above)
Comment (szombi) [reply examples/container/V2containers/GetIngressStatusV2/README.md]: Fixed

File: examples/container/V2containers/GetIngressLoadBalancerConfigV2/README.md (hunk 6 above)
Comment (szombi) [reply examples/container/V2containers/GetIngressLoadBalancerConfigV2/README.md]: Fixed

File: examples/container/V2containers/GetIgnoredIngressStatusErrorsV2/README.md (hunk 5 above)
Comment (szombi) [reply examples/container/V2containers/GetIgnoredIngressStatusErrorsV2/README.md]: Fixed

File: examples/container/V2containers/GetAlbClusterHealthCheckConfigV2/README.md (hunk 4 above)
Comment (szombi) [reply examples/container/V2containers/GetAlbClusterHealthCheckConfigV2/README.md]: Fixed

File: examples/container/V2containers/GetALBAutoscaleConfigurationV2/README.md (hunk 3 above)
Comment (szombi) [reply examples/container/V2containers/GetALBAutoscaleConfigurationV2/README.md]: Fixed

File: examples/container/V2containers/AddIgnoredIngressStatusErrorsV2/README.md (hunk 2 above)
Comment (szombi) [reply examples/container/V2containers/AddIgnoredIngressStatusErrorsV2/README.md]: Fixed

**Review by hkantare — APPROVED** [review]

**Pull request merged**